gradle/rat.gradle

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+apply plugin: "org.nosphere.apache.rat"
+
+rat {
+    excludes = [
+            '**/.github/**',
```

Comment:
Ah, I was already going down a similar path: https://github.com/pravega/pravega/pull/1311
Can you reconcile our two excludes lists and I'll abort my PR?
(I think we can just blanketly exclude anything in a hidden folder)